content/F2016/F16-Lunch-with-Microsoft.md

```diff
@@ -0,0 +1,18 @@
+title: Lunch With Microsoft
+Date: 2016-9-27
+Category: Events
+Tags: social, volunteer, social
+Slug: lunch-with-microsoft
+Author: Shivangi Patwardhan
+Summary: Come join Microsoft for lunch and meet current full-times!
+
+Come join Microsoft for lunch and meet current full time employees! 
+Got class in-between? No worries, drop by anytime between 12:30&ndash;1:30pm to join us for Taco Farm.
```

Comment:
oh, yeah I explained that poorly
I wanted to say that new lines in the code doesn't mean line breaks when it shows up on the website